tests/e2e/Services/Account/AccountCustomServerTest.php

```diff
@@ -218,7 +218,7 @@ public function testCreateMagicUrl(): array
         $this->assertEquals($email, $lastEmail['to'][0]['address']);
         $this->assertEquals($this->getProject()['name'] . ' Login', $lastEmail['subject']);
 
-        $token = substr($lastEmail['text'], strpos($lastEmail['text'], '&secret=', 0) + 8, 64);
+        $token = substr($lastEmail['text'], strpos($lastEmail['text'], '&secret=', 0) + 8, 256);
```

Comment:
Just want to give this attention; is this not breaking? Did token use to be shorter, and now is longer? Can current logic still work with shorter tokens?

Reply:
Only for a few seconds during deployment. We can announce a maintenance window, impact should be minimal.